app/controllers/vacancies_controller.rb

```diff
@@ -17,8 +17,12 @@ class VacanciesController < ApplicationController
   def index
     @filters = VacancyFilters.new(search_params)
     @sort = VacancySort.new.update(column: sort_column, order: sort_order)
-    records = Vacancy.public_search(filters: @filters, sort: @sort).page(page_number).records(includes: [:school])
-    audit_search_event(records, @filters) if @filters.any?
+
+    records = if @filters.any?
```

Comment:
Good idea. I'd propose moving this logic into a new service object to keep on-top of controllers getting too big, perhaps `VacancyFinder`?

Reply:
Sounds like a plan. I'll look into it 👍